### PR TITLE
[11.0][MIG] stock_available: Migration to 11.0

### PR DESCRIPTION
**Pull request by kikopeiro**

### CODE REVIEW

**Comment by fanha99** [comment]
hi, is this module need in v11.0?
there is a field name **Forecasted Quantity** and it show exact the value as field **Available for sale** ?

**Comment by gurneyalex** [comment]
@fanha99 no, forecasted quantity is current stock - reserved + incoming. This module provides current stock - reserved (because sometimes incoming will only be available in a long time in the future)

File: stock_available/views/res_config_settings_views.xml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                                <label for="module_stock_available_mrp"/>
+                                <div class="content-group">
+                                    <div class="mt16" attrs="{'invisible': [('module_stock_available_mrp', '=', False)]}">
+                                        <field name="stock_available_mrp_based_on" class="oe_inline" attrs="{'required':[('module_stock_available_mrp','=',True)]}"/>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+
+                        <!--<div>-->
```
Comment (gurneyalex) [inline stock_available/views/res_config_settings_views.xml:38]: do we need to keep the commented out section?

**Review by gurneyalex — APPROVED** [review]

**Comment by yvaucher** [comment]
OCA Transbot updated translations from Transifex can be squashed together

**Comment by asaunier** [comment]
What do the coverage error messages (``No report found to compare against``) actually mean?
Except the squashing of the OCA Transbot commits, does this PR require some more changes?

**Comment by pedrobaeza** [comment]
Don't worry about that message. It's an old one.

**Comment by pedrobaeza** [comment]
Superseeded by #405

**Comment by pedrobaeza** [comment]
Superseeded by #405

**Pull request closed without merging**